src/vaadin-scroll-container.html

```diff
@@ -61,6 +64,18 @@
           return 'vaadin-scroll-container';
         }
 
+        static get properties() {
+          return {
+            /**
+            * This attribute indicates the scroll direction. Supported values are `vertical`, `horizontal`, `none`. When undefined scrollbars will be shown in both directions.
```

Comment:
*"This property"

Should we now remove the documentation from the "Attribute" section? We don't typically add duplicate descriptions for properties that reflect to attribute.

Reply:
Fixed `attribute` to `property`. 
Looks like we do have duplicates attributes and properties documentation
Eg `disabled` and `invalid` in `vaadin-text-field`
https://cdn.vaadin.com/vaadin-text-field/2.6.0-alpha3/#/elements/vaadin-text-field

Reply:
Removed attribute styling documentation

Reply:
Originally the listing of "state attributes" was meant to be a list of attributes that the component might add by itself. For example, a developer shouldn't add "invalid", "has-value", "focused" etc attributes manually, but those can be added by the component and are needed for styling purposes. The listing you linked shouldn't thus include "disabled" and "readonly" (but it unfortunately does).